<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Oj</class>
 <widget class="QWidget" name="Oj">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1755</width>
    <height>962</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit_in">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>40</y>
     <width>371</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_code">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>811</width>
     <height>911</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>270</y>
     <width>751</width>
     <height>681</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTextEdit" name="textEdit_out2"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_out1"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>代码区域</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>250</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>您的输出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>250</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>正确输出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_compare">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>230</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>比对</string>
   </property>
  </widget>
  <widget class="QComboBox" name="codeType">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>c++</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>python3.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>java</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Golang</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_show">
   <property name="geometry">
    <rect>
     <x>1280</x>
     <y>40</y>
     <width>41</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TestJava.java</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
